pandas_gbq/tests/test_gbq.py

```diff
@@ -1389,6 +1389,30 @@ def test_upload_data_with_missing_schema_fields_raises_error(self):
                        private_key=self.credentials,
                        table_schema=test_schema)
 
+    def test_upload_data_with_timestamp(self):
+        test_id = "21"
+        test_size = 6
+        df = DataFrame(np.random.randn(test_size, 4), index=range(test_size),
+                       columns=list('ABCD'))
+        test_timestamp = datetime.now(pytz.timezone('UTC'))
```

Comment:
Ideally this would be a value rather than `.now`, and that's harder to reproduce, but not a huge issue

Reply:
Thank you for your review. I fixed this.
https://github.com/pydata/pandas-gbq/pull/148/commits/05ac27bad3e6ba33996ee3a59046ca54c22ab5c7